pkgs/top-level/all-packages.nix

```diff
@@ -6030,8 +6030,6 @@ with pkgs;
   moar = callPackage ../tools/misc/moar { };
 
   mobilizon = callPackage ../servers/mobilizon {
-    elixir = elixir_1_14;
```

Comment:
The downside of pinning is, that applications might not be upgraded, even though they would work with the new version.
I'd rather have the responsibility for breakage on the application maintainers (me in this case).

Reply:
Sure, but how would that work in practice?

As in, in a month or two, 1.16 comes out.  Somebody opens a PR to add elixir_1_16 to nixpkgs and make it the default.
- The first problem is that the elixir bump probably won't trigger a rebuild of mobilizon automatically, so we may not know that it's broken.
- The second problem is that, even if we know that it's broken, what do we do then?  Do we poke you on that PR and ask you to contribute to it (probably by bumping the mobilizon version)?  Do we have to re-guess the right invocation to override elixir to 1_15 for mobilizon only, and then maybe discover that this is untestable because the tests broke for some reason?

The alternative is that mobilizon is pinned to 1_15 by default.  When 1_16 gets added to the tree, we can search all-packages.nix for elixir_1_15 and poke the maintainers of those apps to bump the version in their own PRs.

Reply:
@erictapen the problem with leaving it unpinned is that when the update to 1_16 is done, everyone involved in the PR has to remember the individual choices of every package maintainers (mobilizon maintainers prefer it unpinned and are ok with breakage). Someone might object to the PR by seeing ofborg not passing and asking to wait on the maintainers for comment / feedback.
Having it pinned makes sure that when he maintainers have time they can consciously make the update , but even if they dont have time right away , the applicatiom continues to work.
If you dont feel too strong about not pinning, im very much in favor of pinning.

Reply:
I see both your points, pushed the pin!

Reply:
Thank you for being flexible on this !